tsl/src/chunk_copy.c

```diff
@@ -732,6 +900,50 @@ chunk_copy_stage_attach_chunk(ChunkCopy *cc)
 	ts_cache_release(hcache);
 }
 
+static void
+chunk_copy_stage_attach_compressed_chunk(ChunkCopy *cc)
+{
+	char *cmd;
+	const char *chunk_name;
+	const char *compressed_chunk_name;
+	PGresult *res;
+	DistCmdResult *dist_res;
+
+	if (!ts_chunk_is_compressed(cc->chunk))
+		return;
+
+	chunk_name = psprintf("%s.%s",
+						  quote_identifier(cc->chunk->fd.schema_name.data),
+						  quote_identifier(cc->chunk->fd.table_name.data));
+
+	compressed_chunk_name = psprintf("%s.%s",
+									 quote_identifier(INTERNAL_SCHEMA_NAME),
```

Comment:
hmm shouldnt we use the actual schema of the chunk here in case it isnt in internal schema